shell/platform/fuchsia/flutter/software_surface.cc

```diff
@@ -19,6 +19,7 @@
 #include "third_party/skia/include/core/SkImageInfo.h"
 
 #include "../runtime/dart/utils/inlines.h"
+#include "zx/cpp/fidl.h"
```

Comment:
Nit: This is a system library, so <lib/zx/cpp/fidl.h> is preferable

Reply:
Done.